src/components/Card/components/SbCardContent.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="sb-card__content">
+    <slot></slot>
+    <div v-if="isLoading" class="sb-card__loading">
+      <SbLoading :size="'x-large'" :color="'primary'"></SbLoading>
```

Comment:
size="x-large"

Reply:
color="primary"

Reply:
please, use self-close for components without content: https://v2.vuejs.org/v2/style-guide/?redirect=true#Self-closing-components-strongly-recommended

Reply:
@gustavomelki  i've self-closed the components without content